### ISSUE TITLE
processing absolute path import

### ISSUE THREAD

**Issue by mh-cbon** [clean-css/clean-css-cli#44]
hey,

i have a bunch of css such as

```css
//file: backend/public/app.css
@import url("normalize.css");
@import url("/assets/ui/mustard.css");
@import url("/assets/ui/icons.css");
@import url("/assets/ui/alert-box.css");
@import url("/assets/ui/toggle.css");
@import url("/assets/ui/modal.css");
@import url("/assets/ui/color.css");
@import url("/assets/ui/base.css");
@import url("/assets/ui/btn.css");
@import url("fa/css/solid.min.css");
@import url("fa/css/fontawesome.min.css");
@import url("blue.css");
//...
```
Css files using an absolute path (/assets...) are handled within the router using a virutal urls, their real path is located at, starting from the makefile working dir, `backend/public/../../../ui/public`

I try to build my file using 

```sh
cleancss -O1  -o build/backend/public/app.css backend/public/app.css
```
I end up with,

```sh
cleancss -O1 --skip-rebase  -o build/backend/public/app.css backend/public/app.css
ERROR: Ignoring local @import of "/assets/ui/mustard.css" as resource is missing.
ERROR: Ignoring local @import of "/assets/ui/icons.css" as resource is missing.
ERROR: Ignoring local @import of "/assets/ui/alert-box.css" as resource is missing.
ERROR: Ignoring local @import of "/assets/ui/toggle.css" as resource is missing.
ERROR: Ignoring local @import of "/assets/ui/modal.css" as resource is missing.
ERROR: Ignoring local @import of "/assets/ui/color.css" as resource is missing.
ERROR: Ignoring local @import of "/assets/ui/base.css" as resource is missing.
ERROR: Ignoring local @import of "/assets/ui/btn.css" as resource is missing.
```

Is there some switches i can enable right now to pass that issue ? Maybe the cli can benefit a flag to perform pattern matching rebasing, something like `--rebase="/assets to ../../../ui/public"` 

cheers

**Comment by jakubpawlowicz** [comment]
First of all it should not be an error but a warning instead, but that's on `clean-css` module side so I need to fix it there first. 

Regarding dealing with those paths, I have mixed feelings about dealing with absolute paths in general. I used to do some magic to support them in 3.x, then I removed it in 4.x as they proved to be to complex to resolve when it comes to dealing with absolute imports in absolute imports :exploding_head:, then there's sometimes a request to add support for them, but I personally no longer find use for absolute paths. 

Thoughts?
